src/poliastro/spacecraft/__init__.py

```diff
@@ -0,0 +1,38 @@
+from astropy import units as u
+
+
+class Spacecraft:
+    """
+    Class to represent a Spacecraft.
+    """
+
+    @u.quantity_input(A=u.km ** 2, C_D=u.one, m=u.kg)
+    def __init__(self, A, C_D, m, **metadata):
+        """
+        Constructor
+
+        Parameters
+        ----------
+        A: ~astropy.units.Quantity
+        Area of the spacecraft (km^2)
```

Comment:
I think docstring parameters description lines should be indented.

Reply:
Should we ask for `A` and `C_D`, or the ballistic coefficient directly? I think BC is easier to estimate

Reply:
On second thought, it's good that we have the separate components, because some perturbations will make use of each of them (for example, atmospheric drag will need `C_D`).